Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.12-alpine
```

Comment:
Consider using a specific version tag for the Python image instead of `3.12-alpine` to ensure consistent builds. For example, `3.12.0-alpine`.